feedback-mailer/feedback-mailer.gs

```diff
@@ -0,0 +1,210 @@
+const STUDENT_EMAIL = "Student Email ID";
+
+const ALLOTTED_PROJECT = "Which project is allotted to the student?";
+const DISCUSSED_PROJECT = "Which project did the student discuss?";
+
+const VIDEO_STATUS = "Did the student switch on the video?";
+const VIDEO_DURATION = "Duration of the video";
+const PROJECT_EXPLANATION_SCRIPT = "Do you think student is reading from somewhere (like screen, paper etc.) or memorized?";
+const SPEECH_FLOW = "How is the flow of the speech?";
+const GENERAL_FEATURES_EXPLANATION = "Rate the student's explanation on project general features";
+const TECH_STACK_EXPLANATION = "Rate the student's explanation on tech stack";
+const CONTRIBUTION_EXPLANATION = "Rate the student's explanation on his/her contribution";
+
+/**
+ * IDEAL Answers
+ */
+const VIDEO_STATUS_IDEAL = "Yes";
+const VIDEO_DURATION_IDEAL = "1 - 3 min";
+const PROJECT_EXPLANATION_SCRIPT_IDEAL = "Memorized";
+const SPEECH_FLOW_IDEAL = "Good";
+const GENERAL_FEATURES_EXPLANATION_IDEAL = "Good";
+const TECH_STACK_EXPLANATION_IDEAL = "Good";
+const CONTRIBUTION_EXPLANATION_IDEAL = "Good";
+const IDEAL_SCORE = 7;
+
+
+/**
+ * Template Default texts
+ */
+
+const FEEDBACK_STUDENT_NAME = "Student Name";
+const FEEDBACK_TEXT = "Feedback Text";
+const FINAL_TEXT = "Final Text";
+
+const MESSAGE_TEMPLATE = `
+Dear ${FEEDBACK_STUDENT_NAME},
+
+We have reviewed your Project Explanation Video and below is the Feedback:
+
+${FEEDBACK_TEXT}
+
+${FINAL_TEXT}
+
+Thanks & Regards,
+The 10x Academy`;
+
+const LINE_BREAK = ".\n";
+const PROJECT_MISMATCH_PRE_TEXT = "Please discuss about the allotted project while explaining. Your allotted project is ";
+const PROJECT_MISMATCH_POST_TEXT = ", but you have discussed about ";
+const VIDEO_STATUS_NON_IDEAL = ". You have not switched on your video, Please switch on your video while project explanation";
+const VIDEO_DURATION_NON_IDEAL_PRE_TEXT = ". Your video duration is ";
+const VIDEO_DURATION_NON_IDEAL_MID_TEXT = ", The ideal video duration is ";
+const PROJECT_EXPLANATION_SCRIPT_NON_IDEAL = ". It feels like you are reading from somewhere (like screen, paper etc.). Practice memorizing your script so, it feels more authentic";
+const SPEECH_FLOW_NON_IDEAL = ". We have observed that your are getting stuck often with your flow of speech. Take necessary pauses to have a free flow of speech";
+const GENERAL_FEATURES_EXPLANATION_NON_IDEAL_PRE_TEXT = ". Explanation on Project General Features: ";
+const GENERAL_FEATURES_EXPLANATION_NON_IDEAL_POST_TEXT = " Expected: 2 - 3 Lines. Please improve on this";
+const TECH_STACK_EXPLANATION_NON_IDEAL_PRE_TEXT = ". Explanation on Project Tech Stack: ";
+const TECH_STACK_EXPLANATION_NON_IDEAL_POST_TEXT = " Expected: 2 - 4 Lines. Please improve on this";
+const CONTRIBUTION_EXPLANATION_NON_IDEAL_PRE_TEXT = ". Explanation on your contribution in the Project: ";
+const CONTRIBUTION_EXPLANATION_NON_IDEAL_POST_TEXT = " Expected: 3 - 6 Lines. Please improve on this";
```

Comment:
Please use the following strings.

Project
- The project allocated to you was Contacts manager. But you discussed Laundry cart. Please discuss the correct project.

Video
- You have not switched on your video. Please switch on the video and speak to the camera.

Duration
- Your explanation is too short. Please speak for 1 to 3 minutes.
- Your explanation is too long. Please speak for 1 to 3 minutes.

Recital
- It feels like you are reading from somewhere (like screen, paper etc.). Practice your script multiple times and record only after you feel confident, so that it feels more authentic.

Flow
- We have observed that you are getting stuck often. Write down what you want to speak. Practice it multiple times. You can even recite it to your family members or in front of a mirror. Record after you feel confident about the content.

Features
- Explanation about general features is too long. Interviewer may not give you that much time. Please restrict it to 2 - 3 lines.
- Explanation about general features is too short. Interviewer may not understand what the project is about. Please speak around 2 - 3 lines.
- You have not explained what your application does. Please speak 2 - 3 lines on this.

Tech stack
- Explanation about tech stack is too long. Interviewer may not give you that much time. Please restrict it to 2 - 4 lines.
- Explanation about tech stack is too short. Interviewer may feel that you are technically weak. Please speak around 2 - 4 lines.
- You have not talked about the tech stack. Please speak 2 - 4 lines on this.

Contribution
- Explanation about your contribution is too long. Interviewer may not give you that much time. Please restrict it to 3 - 6 lines. Focus on your technical strengths.
- Explanation about your contribution is too short. Interviewer may feel that you are technically weak. Please speak around 3 - 6 lines. Highlight the areas where you are confident.
- You have not talked about your contribution. Please speak 3 - 6 lines on this.

Reply:
Added These Strings